src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -64,7 +64,7 @@ public function __construct(bool $debug = true, string $charset = null, $fileLin
      *
      * @return static
      */
-    public static function register($debug = true, $charset = null, $fileLinkFormat = null)
+    public static function register(bool $debug = true, string $charset = null, string $fileLinkFormat = null)
```

Comment:
`$fileLinkFormat` can also be an instance of `FileLinkFormatter`

Reply:
Excellent throw. 🏀